components/Service/Service.jsx

```diff
@@ -0,0 +1,136 @@
+import { gsap } from 'gsap'
+import Image from 'next/image'
+import { useEffect, useState } from 'react'
+
+import ServiceCard from '@/components/Service/ServiceCard'
+
+import { navigationList } from '@/data/service'
+import useConvert from '@/hooks/useConvert'
+import { pxToVw } from '@/utils/style'
+
+const Service = () => {
+  const { isSp } = useConvert()
+  const [tl] = useState(gsap.timeline({ paused: true }))
+  const [pageCount, setPageCount] = useState(0)
+
+  const nextPage = () => {
+    tl.play()
+    setPageCount(++pageCount)
+  }
+  const backPage = () => {
+    tl.reverse()
+    setPageCount(--pageCount)
+  }
+
+  const jumpPage = (index) => {
+    setPageCount(index)
+    tl.pause(index * 0.8)
+  }
+
+  useEffect(() => {
+    const items = gsap.utils.toArray('div.service-card-element')
+
+    items.forEach((item, index) => {
+      gsap.set(item, { x: isSp ? 10 * index : 30 * index })
+      if (index !== 4) {
+        tl.to(item, {
+          x: -100,
+          y: 1,
+          opacity: 0,
+          duration: 0.4,
+        })
+          .to(items[index + 1], {
+            x: 0,
+            duration: 0.4,
+          })
```

Comment:
@Shu1512 
左に消えるアニメーションと左にずらすアニメーションを一緒に行った方がいいのではないかな
```suggestion
          }, '<')
```